login-workflow/example2/src/screens/new-architecture-test-screens/AccountDetailsScreenBaseTest.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { AccountDetailsScreenBase, AccountDetailsScreenProps } from '@brightlayer-ui/react-auth-workflow';
+import { useNavigate } from 'react-router';
+
+export const AccountDetailsScreenBaseTest: React.FC<AccountDetailsScreenProps> = () => {
+    const navigate = useNavigate();
+
+    const nameValidator = (name: string): boolean => {
```

Comment:
You can simplify this logic as:
`const nameValidator = (name: string): boolean => name.length < 3;`